ui/themes/app.js

```diff
@@ -115,6 +115,7 @@ darkTheme = {
       root : { marginLeft : darkTheme.spacing(1), },
       indicator : {
         height : 3,
+        backgroundColor : "#00B39F",
```

Comment:
nice work 👏 here

Reply:
Reference to a theme color is ideal.

Reply:
Thanks @leecalcote. This would not be possible in the app.js file unless I define the light theme colour before the dark theme. However, this is possible in other files which use the theme. You can check the changes I made in the confirmationModal.js.

Reply:
@meshery/meshery-ui-maintainers any counsel here?

Reply:
Note - This is the file where we are adding all our theme colors, which are then referenced across the Meshery and Extensions. // @leecalcote